toxcore/group_chats.c

```diff
@@ -4429,6 +4429,8 @@ static int peer_add(Messenger *m, int groupnumber, IP_Port *ipp, const uint8_t *
             return -1;
     }
 
+    chat->gcc = tmp_gcc;
```

Comment:
I see now, fixed: https://github.com/JFreegman/toxcore/commit/bfbab7e620f101f2752e4b8acceb5ddfa7204792 It's worth pointing out that if realloc fails, recovery is unlikely.